sdk/modules/ui/cpp/src/base_window.cpp

```diff
@@ -129,9 +129,13 @@ BaseWindow::BaseWindow(const std::string &title, int width, int height, RenderMo
     };
     // clang-format on
 
+  #if defined(__APPLE__) && !defined(__linux__)
+    glGenVertexArraysAPPLE(1, &vertex_array_id_);
```

Comment:
Why not call the standard OpenGL functions? Do they not exist on MacOS?

Maybe in `opengl_api.h` (https://github.com/prophesee-ai/openeb/blob/main/sdk/modules/ui/cpp/include/metavision/sdk/ui/utils/opengl_api.h) you could define:

```
void glGenVertexArrays(GLsizei n, GLuint *arrays) {
    glGenVertexArraysAPPLE(n, arrays);
}

void glBindVertexArray( 	GLuint array) {
    glBindVertexArrayAPPLE(array);
}
```

Reply:
Addressed in the previous comments.